renku/core/commands/client.py

```diff
@@ -66,41 +59,30 @@ def pass_local_client(
         )
 
     def new_func(*args, **kwargs):
-        ctx = click.get_current_context(silent=True)
-        if ctx is None:
-            client = LocalClient(path=default_path(), renku_home=RENKU_HOME, external_storage_requested=True,)
-            ctx = click.Context(click.Command(method))
-        else:
-            client = ctx.ensure_object(LocalClient)
+        cmd = Command().command(method)
 
-        stack = contextlib.ExitStack()
+        if not ignore_std_streams:
+            cmd = cmd.track_std_streams()
 
-        # Handle --isolation option:
-        if get_git_isolation():
-            client = stack.enter_context(client.worktree())
```

Comment:
What happens to isolation then? If we don't want to support, we should remove the command line option and update the docs.

Reply:
It was in the `Command` class, but I move it around to make it more explicit.

Honestly, `--isolation` is one the features in renku that I don't fully understand (how it works/when it is useful). So I wouldn't feel comfortable making a decision to drop it or not. We could open an issue to investigate.